packages/grid/_modules_/grid/components/containers/GridFooterContainer.tsx

```diff
@@ -1,6 +1,7 @@
 import * as React from 'react';
 import clsx from 'clsx';
 import { unstable_composeClasses as composeClasses } from '@mui/material';
```

Comment:
This looks problematic. We bundle the whole `@mui/material` dependency in dev mode.

Reply:
Fixed https://github.com/mui-org/material-ui-x/pull/3094